AllReadyApp/Web-App/AllReady/Views/Shared/_Layout.cshtml

```diff
@@ -1,6 +1,7 @@
-﻿<!DOCTYPE html>
+﻿﻿@using AllReady.Controllers
+<!DOCTYPE html>
 <html lang="en">
-<head>
+﻿<head>

```

Comment:
Somehow a "ZERO WIDTH NO-BREAK SPACE" character (http://stackoverflow.com/questions/9691771/why-is-65279-appearing-in-my-html) got inserted just before the `<head>` tag which is causing the errant white space at the top of the page you see below.
![image](https://cloud.githubusercontent.com/assets/775229/13209560/d613692a-d8ea-11e5-9ba6-718d15b1cdd2.png)